c++/src/capnp/compat/json.h

```diff
@@ -313,6 +313,28 @@ class JsonCodec::Handler<T, Style::POINTER>: private JsonCodec::HandlerBase {
   friend class JsonCodec;
 };
 
+template <>
+class JsonCodec::Handler<DynamicValue>: private JsonCodec::HandlerBase {
```

Comment:
Instead of specializing this, maybe we should actually implement DynamicValue::as<DynamicValue>()? It would just `return *this;` of course.